exercises/proverb/.meta/src/reference/java/Proverb.java

```diff
@@ -0,0 +1,23 @@
+final class Proverb {
+    private String[] words;
+
+    Proverb(String[] words) {
+        this.words = words;
+    }
+
+    String recite() {
+        if (words.length < 1) return "";
+        String initWord = "",
+               result   = "";
```

Comment:
You could use a [`StringBuilder`](https://docs.oracle.com/javase/8/docs/api/java/lang/StringBuilder.html) instead of a string for `result`. A `StringBuilder` is more efficient when you keep appending to it in a loop :)